src/deploy.py

```diff
@@ -1,8 +1,12 @@
 """module concerns itself with tasks involving branch deployments of projects."""
 
+from pprint import pformat
 from fabric.api import task
-from decorators import requires_aws_stack
-from buildercore import bootstrap
+from decorators import requires_aws_stack, debugtask
+# TODO: import modules rather than functions
```

Comment:
I don't care, so long as it's explicit where the module or function came from. No `from blah import *`